### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Zaïd Schouwey

### DISCUSSION THREAD

**Discussion by zaidschouwey98** [heig-vd-dai-course/heig-vd-dai-course#243] [Show and tell]
https://github.com/zaidschouwey98

**Comment by ludelafo** [comment]
Nice little website, thank you! What did you use to make it? :)

**Reply by zaidschouwey98** [reply]
Stenciljs, a library to generate web components. Really easy to use 👍🏼. And I use Netlify to host it since it's only a static website.

**Reply by ludelafo** [reply]
That's cool, thanks!
